blog/2024-03-04-welcome/index.mdx

```diff
@@ -22,11 +21,12 @@ Spatial SQL is a great way to run scalable operations on tables with vector data
C72kGneNJtkZyPtibHur7fP1SRwQ2wxgn2KbNuzbB1DbhLoTmD4Ngsd6gQME2wy3fdCGjpe4HuGTyxkYkQMw7LqI5aa5%2BcsZDlLAnal1IlSFpOElSTfB%2BkWOGa5to2CPC3SkJ9gbBdc8QtWQv6UlKBz8aVxGRvVkj9%2B9OOTF0Do205uGqQQRpMYMtPpY6R3tX3qQwgk241e%2ByZnLJEdZq8DsjPZbA2bGd0wzYF20Y3Hu6tee7TXdU3Z7fS6kyiXze44lo1Ft%2Fuo113cR7luix5dP%2BgxDNljTJskqLsoz7IygiBECSGG5zpasBhCCgjnYcLKStIKBKICHlMKuWuv2L4NKAeb%2FpFxZBNYmhxwTfqKL7urzEJiCwV9dMVziYoIFfZmyl4v3fn4mP55Yz4eG9MYcPMkKkA%2F9x12eK4ocBpTus8jCAfC6b%2FTXQNzpbw9YPWdwqc9aPffTQeTPt8lyg6CSOn3%2BKqQOIY8FWx%2FGI7mTJ7itdFUuEuCXhzF8aKAB21v37WtfNRXqujkYbRcUOBjW%2FSQxdrvOHLl2rSXpHg1G41E%2BVhL%2FgGvIO5l17%2BpCkevksDBtlAReM1%2F38BMMN9BFFjBdOXU7grbfZgcYVlKo%2B0Ze7tQHt0T2BS4SBhIVaD3BxN6SgRUTa2Q2PXBIt%2FcDDD9MzF9jtnibuLUB79PWVyHyn2iYS9bByQOdjG3tmV15C3AeXg%2Bq6i%2BrImYKwtlJdDrrq%2BrrrnyFQ%2Bd5nKE7sK0swVpUoL7FYbFT%2B9nIptc9ohQwVACdpvGlOVLPpQKRrNly2lwm6dbNlW3USKihmJpqmQHCcsuozysFsEGbAKbnTzHyPblrahGgX%2FxwSyjZAOtd0V1%2Bg7%2BhaZJvO9VFPjDXrMAbLwnORDp5OuK%2Bnl3yZ8zsPe821LLrmFonaT5lsTRG1gkGoXVxlWJT62l110STWzTMzGSfK%2FSbg22pH3GZtMdBG%2FudK4qYEA8LtttkyqI6twpkOoitbo%2BkLc05fGO0yDvPXBZWJ8lD%2B%2B7ohcxeNG8cuGV90naFLHrZgUkyP%2FH3dYHFIh3JJhws0iG5eKz4tIX7BwZRPIqUTUkPnTxiXi0U5QsW%2FIv%2Bojn%2FPnJ03OT9pqTpThLkAQt5SRdd7I5noSvJpbtUws7lNLQDyyH%2BpQS4iCTGDYyqYdZiGxGMbWoF1imH7qGgR3sEpPXaTGmnMWGjvqlIUb%2BUpX%2B2Vmd5M%2F7NArjKvNInKt%2BEXS2FHVplcBAXVX%2B%2BFKoTcFh5dPDWkm7%2FupAzuoBLxL5fTB6lZaHaH64wvK7qIp%2BqoszY1cYPv%2FFmbNydpGvd%2FRgHOrzXDMBd%2FaVeFeV6pd%2B9q0QHZ1bV%2FrkkupYyZ%2FSOimtoQ85La%2Bxk7j6%2BRCBIRWp4G%2FPldFoeejMi0Y89MEaT8OKNL5jbfCDO%2FeLDooaDXj%2FVGCkLr%2Ba6KZjhdRDyMGI%2BgZ2fWbpQWjTgBCEHQsZum2ZeqgzRkioI89kBvGQ7lGMfIzJYI5ejZw7MFmDsf8gjvsjjrPcL9xX0%2FC8W3j9OrV7tNQ8XqOGf2MI72EOtqbI2yeQmLT2YDfJQK1iEAcr5S1oooxiCuWnH7%2BTfd%2F%2F87v1z89%2FEFnjB1xXsGhAQysMbN0IGHMdwsBOsMU8RCjxbB05nmdj12AscA2EceD4zA4Q%2FAC87RBxXQEEyo5cSRlkIBkLojBiRXWTOk9AF%2BpqaqcsyqNe0HqJ9vjZz7Iw2NGZ60vH0XTdcyCeUExXczwwbNyV9BDC87B7AGFrHvgAzzwToppFAtycdgKtQQLlx0yyjaFczRWDd2bvBE%2FACjJ1pxc8HZ4q%2BYzw%2FbZngn2j7DkuX9yrXAuDrGiszG9w5bK6lFhpmlrf7JProjaXjJCG25L0Qw6wO7k65sXaV1X4xyxP7yJZjZAukRPWOk4%2BcPOt0F48evntDz98%2F%2FLbocE5mvKCxeFlEEdZa3AOP4ZQRjafbzhYvSmMbjmW57muCxaCw1Antu%2BZFkYmsoF9x0MspNgxPdcn2HCxTnzL9Bh1Dc8NiOv5CIXN6H9uNn9uNi3a59hsXE35Jc3TJI1azXebraaj9TVUrZrvWSRqa0Sc29YnyNp6%2B7c83byeH4NV6No5kZ4RBIHvYtvHgRc62EC2EYQQ5PkmMl3XZLZJMIMIkATYDrDDQsxsBn6cGCZYoa2L44LBbYs7yTa%2FbtGk7I3pHy8afQ5S2osfQ0pGnAXWPMeyXVArrGHLGx57%2FOkxJHO%2Fe49x%2FBrFB1%2BheJ%2BM8X2T5E8rsNOll0MB1PxXdepPK4GPTIIPLxF07wtUTrg63uTdE4EJOUDneHwq8wZ5T7OqHEyvlLdT7tXhRfzdjqpM%2BcEafBVlg3cVTVN54%2B4UqnEMtXvHYIhkHkUSJY4huNWCc%2Bk30HXGP4THR%2BCbDGqIYB9BaCLAIYJzBKHaEIfg7iH4u2qFnz15xmGSMNrscqaUqcKPuhTxd7Lf%2FV1Jc3EoLP6WVLTJVI3XjldJSXKucgDAtxu%2B7NXpPOhJfdDIU65jR%2BPTVnVgB%2BmMBgNwvHV7AsgHv%2B4NenMtObzpKSWNRPAtSGhb6w39UZYpGcnJlsE6FJW6tpOsJs3mP%2F0XgQ0B9n5%2BPiu4B8n4TBTNgX9NU6b87DIRqglY7DW%2FsiOijBdM2Er1d7t11MJzSnKfyLGalLH9S%2BIuv4KsKBxnWqzjTTVqF4vzzECZqy55njcE4D8TdZKz%2F%2B6inG2htZhcXU%2BaK0dVHytK%2FiovG8HLnuQx%2FJLbMbw0R3uXJGCTG3USFU93cfwi4HdaJlehMKr%2FAfwsDPLoPQAA"
```

Comment:
maybe we can add a helper in fused-docs that generates the URLs so we don't need to embed these opaque blobs; obviously this would not be needed when we can embed by Github URL or short link.